### PR TITLE
Hero Changes

### PR DESCRIPTION
**Pull request by nathanakers**
mobile and desktop changes
+ logo layout
+ hero bgs
+ deleted old splash designs

Note in the design: the fonts are Helvetica because it's a local screenshot.
screen shot 2017-12-08 at 12 56 58 am

### CODE REVIEW

File: public_html/assets/css/styles.css [hunk 1]
```diff
@@ -78,14 +78,10 @@ input:focus {background-color:transparent;}
...
-.bg-img.bg-img--active {z-index:-11;}
-.bg-img.bg-img--prev {z-index:-12;}
-
-@media screen and (max-width:959px) and (orientation:portrait), screen and (max-width:767px) {
-    .bg-img {background-image:none;}
-}
+.bg-rotate {position:fixed; top:0; right:0; bottom:0; left:0; z-index:-20;}
+.bg-rotate__img {position:absolute; top:0; left:0; bottom:0; width:100%; min-width:160vh; z-index:-13; background:#fff none center center no-repeat; background-size:contain;}
+.bg-rotate__img.bg-rotate__img--active {z-index:-11;}
+.bg-rotate__img.bg-rotate__img--prev {z-index:-12;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: using a naming system called BEM, I renamed this section better

File: public_html/assets/css/styles.css [hunk 2]
```diff
@@ -497,6 +489,8 @@ body.work-page .nav-wrap hr.title-underline {display:none;}
 }
 
 @media screen and (max-width:959px) {
+    .bg-rotate__img {min-width:0; background-size:cover; background-position:center !important;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: at this size, the background images are centered.

File: public_html/assets/js/scripts.js [hunk 3]
```diff
@@ -164,7 +166,8 @@ $(document).ready(function(){
             logoImgHeight = logoImg.height();
             logoGroupWidth = logoGroup.width();
             logoGroupOffset = logoGroup.offset().left;
-            logoGroupOffsetRight = logoGroupOffset + logoGroupWidth + 30;
+            welcomeSectionDesktopWidth = logoGroupOffset + logoGroupWidth + 50;
```
Comment (nathanakers) [inline public_html/assets/js/scripts.js]: offcentering the background images to -50px from the white background

Comment (nathanakers) [inline public_html/assets/js/scripts.js]: white background width is 50px wider than logo

File: public_html/assets/js/scripts.js [hunk 4]
```diff
@@ -164,7 +166,8 @@ $(document).ready(function(){
...
+            backgroundImageOffset = welcomeSectionDesktopWidth - 200;
```
Comment (nathanakers) [inline public_html/assets/js/scripts.js]: this brings the background to be near the white background, but tucks underneath -150px.

File: public_html/assets/css/styles.css [hunk 5]
```diff
@@ -78,15 +78,11 @@ input:focus {background-color:transparent;}
...
-.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20;}
-.bg-img {position:absolute; top:0; left:0; bottom:0; width:100%; background:#fff none center center no-repeat; z-index:-13; background-size:contain;}
+.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20; background-color:#fff;}
+.bg-img {position:absolute; top:0; right:0; bottom:0; width:52.5%; z-index:-13; background:transparent none center center no-repeat; background-size:cover;}
 .bg-img.bg-img--active {z-index:-11;}
 .bg-img.bg-img--prev {z-index:-12;}
 
-@media screen and (max-width:959px) and (orientation:portrait), screen and (max-width:767px) {
-    .bg-img {background-image:none;}
-}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: moved this to the media queries section

File: public_html/assets/css/styles.css [hunk 6]
```diff
@@ -78,15 +78,11 @@ input:focus {background-color:transparent;}
 /* ---------------------------------
     HOME BG (z-index: -13, -12, -11)
 ------------------------------------ */
-.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20;}
-.bg-img {position:absolute; top:0; left:0; bottom:0; width:100%; background:#fff none center center no-repeat; z-index:-13; background-size:contain;}
+.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20; background-color:#fff;}
+.bg-img {position:absolute; top:0; right:0; bottom:0; width:52.5%; z-index:-13; background:transparent none center center no-repeat; background-size:cover;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: width is just barely non-centered, which is similar-ish to the slight rectangular images.

File: public_html/assets/css/styles.css [hunk 7]
```diff
@@ -221,10 +217,10 @@ section {padding-top:100px; padding-bottom:100px; background-color:#fff; positio
...
 section#welcome .content-wrap {position:fixed; top:45%; left:50%; transform:translate(-50%,-50%) !important;}
-section#welcome .logogroup {font-size:16px; width:auto; margin:0 auto; position:relative; z-index:4; text-align:center;}
-section#welcome .logo {margin:0 auto; width:16em; margin-bottom:2.5em;}
-section#welcome .logo-name {font-size:1.5em; padding:0.45em 0; letter-spacing:0.5em; white-space:nowrap; margin:0 auto 0.25em; background-color:transparent; color:#000;}
-section#welcome .logo-subtext {font-size:1.7em; font-weight:normal; letter-spacing:0.12em; color:#000; margin-bottom:0.75em;}
+section#welcome .logogroup {font-size:15px; width:45%; margin:0 10px; position:relative; z-index:4; text-align:center; display:inline-block;}
+section#welcome .logo {margin:0 auto; width:14em; margin-bottom:1.5em;}
+section#welcome .logo-name {font-size:1.5em; padding:0.45em 0; letter-spacing:0.67em; white-space:nowrap; margin:0 auto 1.5em; background-color:transparent; color:#000; font-weight:normal;}
+section#welcome .logo-subtext {font-size:1.7em; font-weight:bold; letter-spacing:0.12em; color:#000; margin-bottom:0.75em;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: logo design updates

File: public_html/assets/css/styles.css [hunk 8]
```diff
@@ -483,20 +479,20 @@ body.work-page .nav-wrap hr.title-underline {display:none;}
 /* -------------------------------------------------------------------------------------------------------------
     MEDIA STYLES
 -------------------------------------------------------------------------------------------------------------*/
-@media screen and (min-width:960px) {
-    section#welcome .logogroup {display:inline-block; margin:0 10px;}
-}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: moved this above

File: public_html/assets/css/styles.css [hunk 9]
```diff
@@ -483,20 +479,20 @@ body.work-page .nav-wrap hr.title-underline {display:none;}
...
-
 @media screen and (min-width:768px) {
     section#client-compliments .container12:last-of-type .column6 {margin-bottom:0;}
 }
 
 @media screen and (max-width:1160px) {
+    section#welcome .logogroup {font-size:14px;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: at slightly smaller screens, the logo group shrinks slightly using font-size changes. this helps it fit a tad better in the shrinking white space

File: public_html/assets/css/styles.css [hunk 10]
```diff
@@ -509,7 +505,7 @@ body.work-page .nav-wrap hr.title-underline {display:none;}
     header#navbar nav ul li {padding:0 12px;}
     header#navbar nav a svg {height:1.1em; width:1.1em; margin-top:-2px;}
 
-    section#welcome .logogroup {font-size:14px;}
+    section#welcome .content-wrap {display:none;}
```
Comment (nathanakers) [inline public_html/assets/css/styles.css]: hide logo on medium and smaller screens

File: public_html/includes/functions.php [hunk 11]
```diff
@@ -104,15 +104,7 @@
...
-    $bgImages .= '
-    <div class="bg-img bg-img--item-' . $i . '">
-        <style>
-            .bg-img--item-' . $i . ' {
-                background-image: url("/assets/img/BGS/' . $currentBg['filename'] . '");
-                background-color: ' . $currentBg['hexcolor'] . ';
-            }
-        </style>
-    </div>';
+    $bgImages .= '<div class="bg-img bg-img--item-' . $i . '" style="background-image: url(\'/assets/img/BGS/' . $currentBg['filename'] . '\');"></div>';
```
Comment (nathanakers) [inline public_html/includes/functions.php]: with no hexcolor, i just made this prettier

File: public_html/includes/site-data.php [hunk 12]
```diff
@@ -25,19 +25,15 @@
...
     1 => array(
         'filename' => 'bb-splash-biossance.jpg',
-        'hexcolor' => '#fff2eb',
     ),
     2 => array(
         'filename' => 'bb-splash-naya.jpg',
-        'hexcolor' => '#dcdcdc',
     ),
-	3 => array(
+    3 => array(
```
Comment (nathanakers) [inline public_html/includes/site-data.php]: these are spaces that replace rogue tabs

**Comment by carolinapistone** [comment]
Magnificent, wonderful, great job!

**Pull request merged**